public/docs/dart/latest/guide/setup.jade

```diff
@@ -1,58 +1,166 @@
-extends ../../../ts/_cache/guide/setup.jade
-
-block includes
-  include ../_util-fns
-  - var _prereq = 'the Dart SDK';
-  - var _playground = 'repository';
-  - var _Install = 'Get';
-  //- npm/pub commands
-  - var _install = 'get';
-  - var _start = 'serve';
-
-block qs-seed
-  :marked
-    The <live-example name="quickstart">QuickStart project</live-example> can 
-    conveniently be used to seed new projects. It contains the following core files:
-
-block core-files
-  +makeTabs(`
-      quickstart/ts/app/app.component.ts,
-      quickstart/ts/app/main.ts,
-      quickstart/ts/index.html,
-      quickstart/dart/pubspec.yaml,
-      quickstart/ts/styles.css`,
-    ',,,,quickstart',
-    `app/app.component.ts,
-    app/main.ts,
-    index.html,
-    pubspec.yaml,
-    styles.css (excerpt)`)
+include ../_util-fns
+- var _prereq = 'the Dart SDK';
+- var _playground = 'repository';
+- var _Install = 'Get';
+//- npm/pub commands
+- var _install = 'get';
+- var _start = 'serve';
```

Comment:
These local variable definitions are not being used. Unless you think that we can "re-integrate" with TS, we might as well delete lines 2 - 7.